react/TextField/TextField.test.js

```diff
@@ -9,14 +9,11 @@ import {
   scryRenderedDOMComponentsWithClass
 } from 'react-dom/test-utils';
 import { createRenderer } from 'react-test-renderer/shallow';
-import SyntheticEvent from 'react-dom/lib/SyntheticEvent';
```

Comment:
`react-dom/lib/SyntheticEvent` is no longer available, but it seemed like the assertion it was being used for was arguably overkill anyway.